sidebar/js/index.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import Sidebar from "./components/Sidebar";
+
+// function index() {
```

Comment:
here is some more commented out code that should be removed

Reply:
Oops missed it :)